frontend/src/components/regions/CardDetails/Presentation.js

```diff
@@ -168,7 +168,7 @@ export default ({
             showUpdateProjectLinkForm={showUpdateProjectLinkForm}
             linkSubmission={linkSubmission}
             formErrors={formErrors}
-            showAddReviewButton={showAddReviewButton}
+            getShowAddReviewButton={getShowAddReviewButton}
```

Comment:
This is a bug.

The ProjectDetails component says
```
{showAddReviewButton && (
        <CardButton
          label="Add Review"
          startIcon={<RateReviewRoundedIcon />}
          onClick={handleClickAddReview}
        />
      )}
```

It expects that `showAddReviewButton` is handed in as a prop. But now that prob will always be undefined. That means that no more review buttons will show up for anyone ever.  

Function arguments matter. If you dont pass in the right props you wont get the right behavior.  

Please do not edit the `ProjectDetails` component. Please just pass it the information that it needs. I would recommend calling the `getShowAddReviewButton` function somewhere in this code.